Add HelloSnmpAgent example

mattjg908 opened the pull request:
Adds a simple example app and README which makes a Nerves device act as an `SNMP Agent`.

fhunleth commented:
I tried it on a Raspberry Pi Zero and it worked!

I made some minor edits and I'm letting CI compile it on all supported platforms. This takes a while, so I'll probably merge tomorrow. 

Thanks for sending this PR and sorry for the wait.

fhunleth commented:
I manually squashed and merged it. See 91f5e9f4df8f26481bef490c28152b489bfe00dd. 

It was super close to letting you control the GPIO via snmpset's so I added that. You may also notice that I fixed some markdown lint warnings in the README.md. Not sure if it renders perfectly, but it feels close enough and I didn't want to delay merging.

mattjg908 commented:
> I manually squashed and merged it. See [91f5e9f](https://github.com/nerves-project/nerves_examples/commit/91f5e9f4df8f26481bef490c28152b489bfe00dd).
> 
> It was super close to letting you control the GPIO via snmpset's so I added that. You may also notice that I fixed some markdown lint warnings in the README.md. Not sure if it renders perfectly, but it feels close enough and I didn't want to delay merging.

Awesome, thanks @fhunleth and @jeanparpaillon 🎉
